tools/distpartitioning/dataset_utils.py

```diff
@@ -12,14 +12,31 @@
 import torch.distributed as dist
 from gloo_wrapper import alltoallv_cpu
 from utils import (
-    DATA_TYPE_ID,
     generate_read_list,
     get_gid_offsets,
     get_idranges,
     map_partid_rank,
-    REV_DATA_TYPE_ID,
 )
 
+DATA_TYPE_ID = {
```

Comment:
what's the difference better this DATA_TYPE_ID and utils.DATA_TYPE_ID, why do you need to create a new one?